platform/ios/src/MGLMapView.h

```diff
@@ -226,6 +226,15 @@ IB_DESIGNABLE
 - (IBAction)reloadStyle:(id)sender;
 
 /**
+ A control indicating the scale of the map.
+ 
+ The scale bar becomes visible when a user interacts with the
+ map and fades out when the interaction stops. It’s positioned
+ in the upper-left corner.
+ */
+@property (nonatomic, readonly) UIView *scaleBarView;
```

Comment:
I see that you've renamed the class to MGLScaleBar. Just to clarify, my comment above was meant to point out that UITabBar is a "bar" but almost nothing else is. Regardless, I'm comfortable with either name. 👍